README.md

```diff
@@ -87,6 +87,21 @@ vars:
     youtube_analytics_<default_source_table_name>_identifier: your_table_name 
 ```
 
+#### Unioning Multiple Youtube Analytics Connections
```

Comment:
actually could you move this to be the first subsection under the `(Optional) Step 5: Additional configurations` section (so before the `Change the build schema` subsection)?